index.html

```diff
@@ -16,16 +16,22 @@
     <div class="section no-pad-bot">
       <div class="container">
         <br><br>
-        <h1 class="header center teal-text text-lighten-2">Sherpa Clinical</h1>
-        <div class="row center">
-          <h5 class="header col s12 light">Making Connections to Power New Treatments</h5>
-          <h5 class="header col s12 light">Our goal is to improve the health of patients and advance medical science by connecting people with clinical studies and researchers</h5>
+        <div class="row pageHeaderRow">
+          <div class="col offset-s1 s5">
+            <h1 class="pageHeader header light">Sherpa Clinical</h1>
+          </div>
+          <div class="col s3">
+            <h5 class="header light">Making connections to Power New Treatments</h5>
+          </div>
+        </div>
+        <div class="row">
+          <h2 class="header col offset-s1 s7 light">Our goal is to improve the health of patients and advance medical science by connecting people with clinical studies and researchers</h5>
         </div>
         <br><br>
 
       </div>
     </div>
-    <div class="parallax"><img src="background1.jpg" alt="Sherpa Clinical Studies"></div>
+    <div class="parallax"><img style="width:100%" src="\img\background4-cropped.jpg" alt="Sherpa Clinical Studies"></div>
```

Comment:
Oops - thought I had moved that over to CSS already, that was my mistake. Was working on this during my lunch break. Reject this request, I'll put some more detailed work in to it tonight and make sure it works on mobile